index.html

```diff
@@ -1,21 +1,22 @@
 <!--
-    Copyright (c) 2020, RTE (http://www.rte-france.com)
+    Copyright (c) 2024, RTE (http://www.rte-france.com)
     This Source Code Form is subject to the terms of the Mozilla Public
     License, v. 2.0. If a copy of the MPL was not distributed with this
     file, You can obtain one at http://mozilla.org/MPL/2.0/.
 -->
+
 <!DOCTYPE html>
 <html class="singlestretch-parent" lang="en" translate="no">
     <head>
-        <base href="%PUBLIC_URL%/" />
+        <base href="/" />
         <meta charset="utf-8" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0" />
         <meta name="google" content="notranslate" />
-        <link rel="icon" href="favicon.ico" />
+        <link rel="icon" href="/favicon.ico" />
         <title>GridStudy</title>
     </head>
     <body class="singlestretch-parent singlestretch-child">
-        <noscript>You need to enable JavaScript to run this app.</noscript>
```

Comment:
keep this line to have a message when the navigator has scripts disable

Reply:
Done